common/cosmwasm-smart-contracts/vesting-contract/src/lib.rs

```diff
@@ -46,6 +49,46 @@ impl PledgeData {
     }
 }
 
+#[allow(clippy::derive_partial_eq_without_eq)]
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+pub enum PledgeCap {
+    Percent(Percent),
+    Absolute(Uint128), // This has to be in unym
+}
+
+impl FromStr for PledgeCap {
+    type Err = String;
+
+    fn from_str(cap: &str) -> Result<Self, Self::Err> {
+        let cap = cap.replace('_', "").replace(',', ".");
+        match Percent::from_str(&cap) {
+            Ok(p) => {
+                if p.is_zero() {
+                    warn!("Pledge cap set to 0%, are you sure this is right?")
```

Comment:
I know  the purpose of this PR was just to merge branches, but I'd just remove those `warn!` calls (I guess in another PR then?)